### PR TITLE
Graduate Step Slider

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Merges step slider functionality into the pinch slider and graduated from experimental. Cleaned up excessive editor scripts and prefabs

## Changes
- Fixes: #9602 (#9401, #8881), #9606, #9172

### CODE REVIEW

**Comment by RogPodge** [comment]
Most of the functional code up to date. Updated scenes and gifs demonstrating the graduation coming soon, as well as tests for the new snapping/touch slider elements

File: Assets/MRTK/SDK/Features/UX/Scripts/Sliders/PinchSliderInspector.cs [hunk 1]
```diff
@@ -9,20 +9,21 @@
...
+
         private static GUIStyle labelStyle;
 
-        public override void OnInspectorGUI()
-        {
-            if (target != null)
-            {
-                InspectorUIUtility.RenderHelpURL(target.GetType());
-            }
+        //public override void OnInspectorGUI()
```
Comment (davidkline-ms) [inline Assets/MRTK/SDK/Features/UX/Scripts/Sliders/PinchSliderInspector.cs:18]: was this meant to be completely removed?

File: Assets/MRTK/SDK/Features/UX/Scripts/Sliders/SliderSounds.cs [hunk 2]
```diff
@@ -66,7 +70,13 @@ private void Start()
             {
                 passNotchAudioSource = gameObject.AddComponent<AudioSource>();
             }
+
```
Comment (davidkline-ms) [inline Assets/MRTK/SDK/Features/UX/Scripts/Sliders/SliderSounds.cs:73]: nit: one blank line seems sufficient :smile:

**Review by davidkline-ms — COMMENTED** [review]
one question to resolve, otherwise looks good

**Comment by RogPodge** [comment]
All up to date with new unit tests. Example gifs coming soon!

**Comment by RogPodge** [comment]
Slider1
Slider2
Slider3

**Review by davidkline-ms — APPROVED** [review]

**Pull request merged**